statsbombpy/sb.py

```diff
@@ -146,18 +154,40 @@ def competition_events(
     return competition_events
 
 
-def frames(
+def _360_frames(
     match_id: int,
-    fmt: str = "dataframe",
     creds: dict = DEFAULT_CREDS,
 ) -> Union[pd.DataFrame, list, dict]:
     if api_client.has_auth(creds) is True:
         frames = api_client.frames(match_id, creds=creds)
     else:
         frames = public.frames(match_id)
+    return frames
+
+
+def frames(
+    match_id: int,
+    fmt: str = "dataframe",
+    creds: dict = DEFAULT_CREDS,
```

Comment:
should we add a flag to exclude 360 frames? I guess some customer would just want shot frames?

Reply:
Shot frames are part of the events. They don't get added to the frames function. This just handles 360 FFs.